docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,50 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+If you have common configuration options you want to apply to all profiles, you have to do this manually per profile.
+See xref:../using/profiles.adoc#setting-global-config-profiles[Setting Global Configuration Options].
+
+The current profile being used is called the active profile.
```

Comment:
This needs a bit more of lead in imo. Also I would try to word things along these lines:

"Unless the global  `--profile` flag is used, Minishift commands are executed against the so-called active profile. You can determine the active profile by using `minishift profile list`.  You can execute single commands against a non-active profile, by using the `--profile` flag, eg `minishift --profile my-profile console --url`.

On top of the  `--profile` flag, there are commands for setting , listing and deleting  the active profile. These commands are described in the following sections."